### ISSUE TITLE
Can't build 3.0.12 .deb packages

### ISSUE THREAD

**Issue by captainark** [FreeRADIUS/freeradius-server#1771]
# Issue type
- [x] Defect - Unexpected behaviour (obvious or verified by project member).

# Defect/Feature description
## How to reproduce issue

Building Freeradius 3.0.12 .deb packages from sources downloaded from http://freeradius.org/download.html fails :
 
```text
dpkg-buildpackage: paquet source freeradius
dpkg-buildpackage: version source 3.0.12+git
dpkg-buildpackage: distribution source unstable
dpkg-buildpackage: source changé par Alan DeKok <aland@freeradius.org>
dpkg-buildpackage: architecture hôte amd64
 dpkg-source --before-build freeradius-server-3.0.12
 fakeroot debian/rules clean
QUILT_PATCHES=debian/patches \
        quilt --quiltrc /dev/null pop -a -R || test $? = 2
Aucun patch retiré
rm -rf .pc debian/stamp-patched
dh_testdir
dh_testroot
rm -f build-arch-stamp build-indep-stamp
rm -f config.cache config.log
rm -f src/freeradius-devel
[ ! -d src/modules/lib ] || rm -fr src/modules/lib || true
[ ! -d src/binary ] || rm -fr src/binary || true
# Add here commands to clean up after the build process.
dh_clean
        rm -f debian/freeradius.substvars
        rm -f debian/freeradius.*.debhelper
        rm -rf debian/freeradius/
        rm -f debian/freeradius-common.substvars
        rm -f debian/freeradius-common.*.debhelper
        rm -rf debian/freeradius-common/
        rm -f debian/freeradius-config.substvars
        rm -f debian/freeradius-config.*.debhelper
        rm -rf debian/freeradius-config/
        rm -f debian/freeradius-utils.substvars
        rm -f debian/freeradius-utils.*.debhelper
        rm -rf debian/freeradius-utils/
        rm -f debian/libfreeradius3.substvars
        rm -f debian/libfreeradius3.*.debhelper
        rm -rf debian/libfreeradius3/
        rm -f debian/libfreeradius-dev.substvars
        rm -f debian/libfreeradius-dev.*.debhelper
        rm -rf debian/libfreeradius-dev/
        rm -f debian/freeradius-dhcp.substvars
        rm -f debian/freeradius-dhcp.*.debhelper
        rm -rf debian/freeradius-dhcp/
        rm -f debian/freeradius-krb5.substvars
        rm -f debian/freeradius-krb5.*.debhelper
        rm -rf debian/freeradius-krb5/
        rm -f debian/freeradius-ldap.substvars
        rm -f debian/freeradius-ldap.*.debhelper
        rm -rf debian/freeradius-ldap/
        rm -f debian/freeradius-rest.substvars
        rm -f debian/freeradius-rest.*.debhelper
        rm -rf debian/freeradius-rest/
        rm -f debian/freeradius-postgresql.substvars
        rm -f debian/freeradius-postgresql.*.debhelper
        rm -rf debian/freeradius-postgresql/
        rm -f debian/freeradius-mysql.substvars
        rm -f debian/freeradius-mysql.*.debhelper
        rm -rf debian/freeradius-mysql/
        rm -f debian/freeradius-iodbc.substvars
        rm -f debian/freeradius-iodbc.*.debhelper
        rm -rf debian/freeradius-iodbc/
        rm -f debian/freeradius-redis.substvars
        rm -f debian/freeradius-redis.*.debhelper
        rm -rf debian/freeradius-redis/
        rm -f debian/freeradius-memcached.substvars
        rm -f debian/freeradius-memcached.*.debhelper
        rm -rf debian/freeradius-memcached/
        rm -f debian/freeradius-yubikey.substvars
        rm -f debian/freeradius-yubikey.*.debhelper
        rm -rf debian/freeradius-yubikey/
        rm -f debian/freeradius-dbg.substvars
        rm -f debian/freeradius-dbg.*.debhelper
        rm -rf debian/freeradius-dbg/
        rm -f debian/*.debhelper.log
        rm -f debian/files
        find .  \( \( \
                \( -path .\*/.git -o -path .\*/.svn -o -path .\*/.bzr -o -path .\*/.hg -o -path .\*/CVS \) -prune -o -type f -a \
                \( -name '#*#' -o -name '.*~' -o -name '*~' -o -name DEADJOE \
                 -o -name '*.orig' -o -name '*.rej' -o -name '*.bak' \
                 -o -name '.*.orig' -o -name .*.rej -o -name '.SUMS' \
                 -o -name TAGS -o \( -path '*/.deps/*' -a -name '*.P' \) \
                \) -exec rm -f {} + \) -o \
                \( -type d -a -name autom4te.cache -prune -exec rm -rf {} + \) \)
        rm -f *-stamp
 debian/rules build
QUILT_PATCHES=debian/patches \
        quilt --quiltrc /dev/null push -a || test $? = 2
Application de radiusd-to-freeradius.diff
patching file Make.inc.in
patching file raddb/radiusd.conf.in
patching file scripts/monit/freeradius.monitrc
Hunk #1 succeeded at 8 with fuzz 1.
patching file raddb/sites-available/control-socket
patching file src/main/radiusd.c

Application de disable-dhcp-bydefault.diff
patching file raddb/all.mk
Hunk #1 FAILED at 8.
1 out of 1 hunk FAILED -- rejects in file raddb/all.mk
Le patch disable-dhcp-bydefault.diff ne s'applique pas proprement (forcez l'application avec -f)
/usr/share/quilt/quilt.make:16: recipe for target 'debian/stamp-patched' failed
make: *** [debian/stamp-patched] Error 1
dpkg-buildpackage: erreur: debian/rules build a produit une erreur de sortie de type 2
```

**Comment by alandekok** [comment]
fixed by commit e5cce0504

**Comment by captainark** [comment]
It works, thanks a lot @alandekok !
